react-debugging.html

```diff
@@ -28,7 +28,7 @@
           }
         }
 
-        drawCard = () => {
+        drawCard = (props) => {
```

Comment:
no need to add props as an argument unless you plan on using it within the method